Jenkins Operator on Openshift.md

```diff
@@ -0,0 +1,185 @@
+Jenkins Operator on Openshift
+
+Due to the way the Openshift specific image for Jenkins works, we have to create a few extra resources and define some new resources as well.

```

Comment:
It would be awesome if we could list all the things that are different from e.g. EKS, all the constraints related with default hardening in OC, etc.

Reply:
Don't think this would be necessary for now.